tests/test_farmit.py

```diff
@@ -56,6 +55,27 @@ def repo(tmp_path, remote_url, request):
     os.chdir(request.config.invocation_dir)
 
 
+@pytest.fixture
+def no_tags_repo(tmp_path, remote_url, request):
```

Comment:
Please make this code DRY. `repo` should call `no_tags_repo`.

Reply:
Seeking a bit of clarification on this item -- could you clarify how one might combine these two methods?

Reply:
we should discuss this

Reply:
Looks like we can if/then around lines 45 and 68, and keep most of the rest of the code the same.

Reply:
Good catch. This is done.